tf-ansible/aws.yaml

```diff
@@ -0,0 +1,119 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+
+imports:
+  - https://cloudify.co/spec/cloudify/6.2.0/types.yaml
+  - plugin:cloudify-terraform-plugin
+  - plugin:cloudify-ansible-plugin
+
+inputs:
+  aws_region_name:
+    type: string
+    description: AWS Region to deploy resources into
+    default: us-west-2
+
+  instance_type:
+    type: string
+    description: EC2 instance type to deploy (e.g., t2.small)
+    default: t2.small
+
+  public_key_content:
+    type: string
+    description: Contents of a public key. The corresponding private key must be in the secret store.
+
+  vpc_cidr:
+    type: string
+    description: CIDR network for the deployed VPC
+
+  subnet_cidr:
+    type: string
+    description: CIDR network for the deployed subnet. Must fall within the VPC CIDR.
+
+  website_message:
+    type: string
+    description: A message to display on the home page of the deployed web server
+    default: Welcome to NGINX, deployed by Cloudify
+
+node_templates:
+
+  terraform:
+    type: cloudify.nodes.terraform
+    properties:
+      resource_config:
+        installation_source: https://releases.hashicorp.com/terraform/1.1.4/terraform_1.1.4_linux_amd64.zip
+  
+  shared_resources:
+    type: cloudify.nodes.terraform.Module
+    properties:
+      resource_config:
+        source:
+          location: aws/shared_resources.zip
```

Comment:
I prefer that we not clutter the repo with binary files. Is it possible to store this in another repository? alternatively we can provide a permalink.